ui/src/components/info-dialog/info-dialog.module.scss

```diff
@@ -0,0 +1,33 @@
+@import 'design-system/variables/colors.scss';
+@import 'design-system/variables/typography.scss';
+
+.content {
+  min-width: 640px;
+  max-width: 100%;
+  padding: 24px 32px;
+  color: $color-neutral-800;
+
+  h1 {
```

Comment:
Some simple styling of HTML elements, let me know if we need more and we can easily extend. The app global styles are a bit limited since we are using CSS modules (making each component define its own styles, rather than using global CSS).